File: 168-excel-sheet-column-title/excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        chars="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
        res=""
        while columnNumber:
            columnNumber-=1
            res+=chars[columnNumber % 26]
            columnNumber//=26
        return res[::-1]

File: 168-excel-sheet-column-title/excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string s="";
        while(n>0)
        {     
              int temp = n%26 ;
              char t;
              if(temp == 0)t = 'Z';
              else t = temp-1+'A';
    
              n=(n-1)/26;
              s=t+s;

        }
        return s;
    }
};